[INFO -- 2018-09-25 07:27:50]
Starting flip_value script
version: 1.1
[INFO -- 2018-09-25 07:27:50]
Initialization strings:
[INFO -- 2018-09-25 07:27:50]
(1) file /tmp/knapsack/ga
[INFO -- 2018-09-25 07:27:50]
initSignal:1
[INFO -- 2018-09-25 07:27:50]
endSignal:3.5
[INFO -- 2018-09-25 07:27:50]
faultModel:2
[EXCEPTION -- 2018-09-25 07:27:52]
gdbException: Cannot access memory at address 0x1ecd401c10e832af
[EXCEPTION -- 2018-09-25 07:27:52]
pythonException: Unable to exit loop in struct fields; Exiting wihtout making a bit flip
[EXCEPTION -- 2018-09-25 07:27:52]
gdbException: Cannot access memory at address 0xab4b0bcc0d58f63
[INFO -- 2018-09-25 07:27:52]
Backtrace BEGIN:
#0  0x0000555555556361 in std::swap<int> (__a=<error reading variable>, __b=@0x5555557855b0: 1) at /usr/include/c++/7/bits/move.h:198
#1  0x00005555555551e7 in crossover (pai=std::vector of length 200, capacity 200 = {...}, mae=std::vector of length -192835483982056328, capacity -192835483982056328 = {...}, filho1=std::vector of length -554851713913976103, capacity -554851713913976103 = {...}, filho2=std::vector of length 200, capacity 200 = {...}) at knapsackga.cpp:80
#2  0x000055555555548d in genetic_algorithm () at knapsackga.cpp:104
#3  0x00005555555558d7 in main () at knapsackga.cpp:137

193	#endif
194	    {
195	      // concept requirements
196	      __glibcxx_function_requires(_SGIAssignableConcept<_Tp>)
197	
198	      _Tp __tmp = _GLIBCXX_MOVE(__a);
199	      __a = _GLIBCXX_MOVE(__b);
200	      __b = _GLIBCXX_MOVE(__tmp);
201	    }
202	

Backtrace END
Memory content before bitflip:0x310xfb0x080x00
Fault Model: Random bit-flip
base address to flip value: 0x55555575f06c 
address max offset: 4
Memory content after  bitflip:0x6c0xa10x9b0x60
frame name: crossover
symbol name: W
symbol filename: knapsackga.cpp
symbol line: 16
value: 1620812140
value address: 0x55555575f06c <W>
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2018-09-25 07:27:52]
Fault Injection Successful after 2.489370822906494s
[INFO -- 2018-09-25 07:27:52]
event type: exit
[EXCEPTION -- 2018-09-25 07:27:52]
exit code: no exit code available
